handlers/twiml/handlers.go

```diff
@@ -253,7 +253,16 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 
 		for _, comp := range msg.Templating().Components {
 			for varKey, varIndex := range comp.Variables {
-				contentVariables[varKey] = msg.Templating().Variables[varIndex].Value
+				value := msg.Templating().Variables[varIndex].Value
+				attType, value := handlers.SplitAttachment(value)
+				attType = strings.Split(attType, "/")[0]
+
+				// not attachment ignore the split, revert to original entire value
+				if attType != "image" && attType != "video" && attType != "application" {
+					value = msg.Templating().Variables[varIndex].Value
+				}
```

Comment:
actually this feels a bit weird and is liable to do the wrong thing if a text variable looks like an attachment... can't we check variable type like in the WhatsApp handler ?

Reply:
Yes, I had found that just after pushing